lib/streamlit/web/server/server.py

```diff
@@ -83,6 +85,7 @@
 UNIX_SOCKET_PREFIX = "unix://"
 
 MEDIA_ENDPOINT: Final = "/media"
+UPLOAD_FILE_ENDPOINT: Final = "/_stcore/upload_file"
```

Comment:
I wonder if this should be something like `UPLOAD_FILE_ENDPOINT_PREFIX` or `UPLOAD_FILE_ROUTE` (or some better name because I'm not really happy with either of those) since the full endpoint contains both a `sessionId` and `file_id`.